pages/task/read/views/models/project.jsx

```diff
@@ -84,15 +84,18 @@ export default function Project({ task, schema }) {
                 <p><strong><Snippet>declarations.pel-holder.question</Snippet></strong></p>
                 <p>{declarations.authority}</p>
                 {
-                  declarationConfirmed(declarations.authority) &&
-                    <Fragment>
-                      <p>
-                        <Snippet>declarations.pel-holder.name</Snippet> {declarations['authority-pelholder-name']}
-                      </p>
-                      <p>
-                        <Snippet>declarations.pel-holder.endorsement-date</Snippet> {declarations['authority-endorsement-date']}
-                      </p>
-                    </Fragment>
+                  declarationConfirmed(declarations.authority)
+                    ? (
+                      <Fragment>
+                        <p>
+                          <Snippet>declarations.pel-holder.name</Snippet> {declarations['authority-pelholder-name']}
+                        </p>
+                        <p>
+                          <Snippet>declarations.pel-holder.endorsement-date</Snippet> {declarations['authority-endorsement-date']}
+                        </p>
+                      </Fragment>
+                    )
+                    : 'No answer provided'
```

Comment:
I meant to put this in each of the two follow up questions.

`declarationConfirmed(declarations.authority)` will still return true if the application has been through a "new" endorsement journey, but the two subquestions will not have values.